src/main/kotlin/no/nav/familie/ef/sak/vedtak/dto/VedtakDto.kt

```diff
@@ -97,8 +119,13 @@ fun Vedtak.tilVedtakDto(): VedtakDto =
                     begrunnelse = this.avslåBegrunnelse,
                     opphørFom = YearMonth.from(this.opphørFom)
             )
+            ResultatType.SANKSJONERE -> Sanksjonert(
+                    resultatType = this.resultatType,
+                    sanksjonsårsak = this.sanksjonsårsak ?: Sanksjonsårsak.INGEN_VALGT,
+                    periode = (this.perioder?.perioder ?: PeriodeWrapper(emptyList()).perioder).fraDomeneForSanksjon(),
```

Comment:
Trenger ikke PeriodeWrapper her
```suggestion
                    periode = (this.perioder?.perioder ?: emptyList()).fraDomeneForSanksjon(),
```

Reply:
Fixed i [87b4db9](https://github.com/navikt/familie-ef-sak/pull/1172/commits/87b4db979f71a0b7180acc19e44fb3898e683a69)